vars/perfRegressionParallelPipeline.groovy

```diff
@@ -131,14 +130,21 @@ def call(Map pipelineParams) {
                                         checkout scm
                                         (testDuration, testRunTimeout, runnerTimeout, collectLogsTimeout, resourceCleanupTimeout) = getJobTimeouts(params, builder.region)
                                         base_versions_list = params.base_versions.contains('.') ? params.base_versions.split('\\,') : []
-                                        // NOTE: master and enterprise will have 1 single version, but for releases,
-                                        // we will choose automatically only the last one
-                                        supportedVersions = supportedUpgradeFromVersions(
-                                            base_versions_list,
-                                            'ubuntu-focal',
-                                            params.new_scylla_repo,
-                                            params.backend
-                                        )
+                                        def new_repo = params.new_scylla_repo
+                                        supportedVersions = ''
+                                        new_params = params.collectEntries { param -> [param.key, param.value] }
+                                        if (new_repo) {
+                                            // NOTE: master and enterprise will have 1 single version, but for releases,
+                                            // we will choose automatically only the last one
+                                            supportedVersions = supportedUpgradeFromVersions(
```

Comment:
> looks like you take only 1 version there not multiple "supportedVersions" .last()

yes, and that is intentional, 1st, because on master (where we mainly run this test) we have always 1 base version, but when running on a release, we will test only the last one, and this is to avoid complexity in the parallel stages we currently have, since it is already a loop of "sub tests", and we don't need to "automatically" run on all base versions IMHO